frontend/packages/console-shared/src/components/Badges.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { Label } from '@patternfly/react-core';
+
+export const TechPreviewBadge: React.FC = () => (
```

Comment:
These should be their own separate components since we want to follow the convention of one Component per file.
If you want to make a sub dir named badges, that's ok.

Reply:
Done.